### PR TITLE
longer forcemerge timeout

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

**Review by ch-kr — APPROVED** [review]
looks good to me. basic question, would it be helpful to also increase the general default timeout?

File: hail_scripts/v02/utils/elasticsearch_client.py [hunk 1]
```diff
@@ -204,4 +204,4 @@ def export_table_to_elasticsearch(
         es.batch.write.refresh // default true  (Whether to invoke an index refresh or not after a bulk update has been completed)
         """
 
-        self.es.indices.forcemerge(index=index_name)
+        self.es.indices.forcemerge(index=index_name, request_timeout=60)
```
Comment (mike-w-wilson) [inline hail_scripts/v02/utils/elasticsearch_client.py:207]: I think the value is in seconds so this won't change the current 1m config. https://elasticsearch-py.readthedocs.io/en/7.10.0/api.html#timeout

**Review by mike-w-wilson — CHANGES_REQUESTED** [review]
I may have looked at the wrong docs but I think we need this to be 3600 to be an hour timeout?

**Review by mike-w-wilson — APPROVED** [review]

**Comment by mike-w-wilson** [comment]
Hana explained the default is currently 10s

**Pull request merged**